### PR TITLE
[docs][7.4] Uncomment APM tagged regions

### PR DESCRIPTION
**Pull request by bmorelli25**
~Blocked by https://github.com/elastic/apm-server/pull/2734.~

OK to merge.

### CODE REVIEW

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Comment by bmorelli25** [comment]
Took a few tries, but this PR is _finally_ green @lcawl :)

A [preview of the changed page](http://stack-docs_529.docs-preview.app.elstc.co/guide/en/elastic-stack/7.4/apm-highlights.html). Want me to remove the coming tag in this PR as well?

File: docs/en/install-upgrade/breaking.asciidoc [hunk 1]
```diff
@@ -31,9 +31,9 @@ Upgrade Assistant provides information about which {dfeeds} need to be updated.
 coming[7.4.0]
 
 This list summarizes the most important breaking changes in APM. 
-For the complete list, go to {apm-server-ref-70}/breaking-changes.html[APM Server 7.0 breaking changes].
+For the complete list, go to {apm-server-ref}/breaking-changes.html[APM Server 7.0 breaking changes].
```
Comment (lcawl) [inline docs/en/install-upgrade/breaking.asciidoc]: In my opinion, it seems a bit strange to call these 7.0 breaking changes in this context:
```suggestion
For the complete list, go to {apm-server-ref}/breaking-changes.html[APM Server breaking changes].
```

**Comment by lcawl** [comment]
Thanks for making this work, @bmorelli25 ! I think it's fine to leave the removal of the "coming" tag for a separate PR after the release occurs, I'll create one now.

**Pull request merged**